build.gradle

```diff
@@ -53,14 +61,15 @@ subprojects {
                  "reactive-streams-tck",
                  "reactive-streams-tck-flow",
                  "reactive-streams-examples"]) {
-        apply plugin: "maven"
         apply plugin: "signing"
+        apply plugin: "publishing"
         apply plugin: "maven-publish"
```

Comment:
Small build cleanup because Gradle `7.x` eliminates the `maven` plugin in favor of `publishing`. Other than DSL updates, the logic is unchanged.